### PR TITLE
Fix silent mode 

### PR DESCRIPTION
**Pull request by joykuotw**
Modify the silent mode cause it is not completely silent, and also the raised TypeError due to missing attribute while using --do_compression option. In addition, make sure to get the precise logging message.

### CODE REVIEW

File: mbutil/util.py [hunk 1]
```diff
@@ -230,10 +236,8 @@ def disk_to_mbtiles(directory_path, mbtiles_file, **kwargs):
                             (?, ?, ?, ?);""",
                             (z, x, y, sqlite3.Binary(file_content)))
                         count = count + 1
-                        if (count % 100) == 0:
-                            for c in msg: sys.stdout.write(chr(8))
-                            msg = "%s tiles inserted (%d tiles/sec)" % (count, count / (time.time() - start_time))
-                            sys.stdout.write(msg)
+                        if (count % 100) == 0 and not silent:
+                            logger.info(" %s tiles inserted (%d tiles/sec)" % (count, count / (time.time() - start_time)))
```
Comment (sgillies) [inline mbutil/util.py]: If anyone is depending on this output in their scripts, the change will break them. I'm not saying that was a good choice, only looking out for users.

@perrygeo what do you think?

Comment (perrygeo) [inline mbutil/util.py]: I'm ok with this change, makes things more consistent. I think the chances of breaking anything is minimal. I can't think why anyone would need to parse that particular output but maybe I'm not being creative enough :-)

Comment (sgillies) [reply mbutil/util.py]: Okay, sure.

**Comment by sgillies** [comment]
@joykuotw thanks! FYI, we don't have a future release scheduled yet.

**Pull request merged**